src/index.ts

```diff
@@ -1066,6 +1073,7 @@ function makeOptionsConfident(
       "determinate-nixd",
       actionOptions.name,
     ],
+    binaryNamesDenyList: actionOptions.binaryNamePrefixes ?? ["nix-expr-tests"],
```

Comment:
Let's make this a constant or something somewhere that's more easily visible / extensible.